java/code/src/com/redhat/rhn/domain/common/CommonFactory.java

```diff
@@ -87,9 +89,12 @@ public static FileList lookupFileList(Long idIn, Org org) {
         Session session = null;
         //look for Kickstart data by id
         session = HibernateFactory.getSession();
-        return (FileList) session.getNamedQuery("FileList.findByIdAndOrg")
```

Comment:
Can't you remove the named query at the same time to avoid forgetting it?

Reply:
done